Yoast/wordpress-seo
# Fix missing meta tags woocommerce

## Summary
Fixes a bug in the RC where not all metatags are rendered for a WooCommerce shop page.

## Test instructions

This PR can be tested by following these steps:

* Set all the social things for the WooCommerce shop page 
* View the source and see all the metatags still works.

Fixes #8485
